git_greppers/utils.sh

```
@@ -1,30 +1,300 @@
+    else
+      echo "$(( seconds_diff / 2592000 )) months ago"
+    fi
+  else
+    # Fallback to formatted date
+    format_date "$date_string"
+  fi
+}
+
+# ANSI color codes
+RED='\033[0;31m'
+GREEN='\033[0;32m'
```

Shellcheck (0.10.0)

[warning] GREEN appears unused. Verify use (or export if used externally).

(SC2034)